.travis.yml

```diff
@@ -1,5 +1,5 @@
-os:
-  - linux
+os: linux
+dist: trusty
```

Comment:
Instead of running on a particular distribution, it will be better if we can use openjdk8 and openjdk11.

Reply:
Happy to do that; I was just following the initial setup by @shubhamsc which was using oraclejdk; but if open to switching to openjdk, I agree that would be better; will update the PR.